packages/create-docusaurus/src/index.ts

```diff
@@ -269,34 +269,32 @@ ${chalk.cyan('Creating new Docusaurus project...')}
   }
 
   const pkgManager = useYarn ? 'yarn' : 'npm';
+  let isInstallSuccessful = true;
   if (!cliOptions.skipInstall) {
     console.log(`Installing dependencies with ${chalk.cyan(pkgManager)}...`);
-
-    try {
-      // Use force coloring the output, since the command is invoked by shelljs, which is not the interactive shell
+    if (
       shell.exec(
         `cd "${name}" && ${useYarn ? 'yarn' : 'npm install --color always'}`,
         {
           env: {
             ...process.env,
+            // Force coloring the output, since the command is invoked by shelljs, which is not the interactive shell
             ...(supportsColor.stdout ? {FORCE_COLOR: '1'} : {}),
           },
         },
-      );
-    } catch (err) {
-      console.log(chalk.red('Installation failed.'));
-      throw err;
+      ).code !== 0
+    ) {
+      isInstallSuccessful = false;
```

Comment:
was wondering if we couldn't abort early here? throw or `exit(1)``

That could make the workflow simpler

Reply:
I don't want it to exit with 1. Installation is an optional step that the user can easily recover from. As long as we have a clear message, I think it's fine

Reply:
then what about exit 0 :D as long as you bail out early the code becomes simpler

Reply:
Ah, I see what you mean😄Yeah, makes sense

Reply:
Thanks, :)